composer/loggers/remote_uploader_downloader.py

```diff
@@ -707,3 +708,7 @@ def upload_file(retry_index: int = 0):
         time.sleep(local_rank * local_rank_stagger)
 
         upload_file()
+
+    for destination in logger.destinations:
+        if isinstance(destination, MosaicMLLogger):
+            destination.log_metadata({'checkpoint_uploaded_time': time.time()})
```

Comment:
should this have a force flush flag?

Reply:
log_metadata always flushes at the end
https://github.com/mosaicml/composer/blob/4e1eb8d20a904510c2c03819f4517c1f12fe7b3a/composer/loggers/mosaicml_logger.py#L164